src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -12,7 +12,7 @@ internal static partial class PathInternal
     {
         // There is only one invalid path character in Unix
         private const char InvalidPathChar = '\0';
-        internal static readonly char[] InvalidPathChars = { InvalidPathChar };
+        internal static readonly char[] InvalidPathChars = GetInvalidPathChars();
```

Comment:
The field could be defined once in `PathInternal.cs`, instead of separate field definitions for .Unix.cs and .Windows.cs (only the methods have to be different).